src/DataGrid.php

```diff
@@ -731,11 +731,22 @@ public function getSortableHandler()
 
 
 	/**
+	 * @param Column  $column
+	 * @return array
+	 * @internal
+	 */
+	public function getSortNext(\Ublaboo\DataGrid\Column\Column $column)
```

Comment:
Špatně jsem to napsal, necháme to takto, nebo bys tu fci přímo nad DataGrid classou pojmenoval jinak? Hádám že necháme.